<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Какой у вас треугольник?</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>321</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_massege">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>181</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Это треугольник!!!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_try_again">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Повторить</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_input">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>271</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_varify_triangle">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>100</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Это треугольник?</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_1_side">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>52</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Сторона 1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2_side">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_1_side">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2_side">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>52</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Сторона 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>52</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Сторона 3</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3_side">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
